front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -34,52 +45,138 @@ const CreateAndEditProjectsContainer = styled.div`
+    return 'published';
+  } else if (pathname.includes('/admin/projects/draft')) {
+    return 'draft';
+  } else if (pathname.includes('/admin/projects/archived')) {
+    return 'archived';
+  } else {
+    return 'your-projects';
+  }
+};
+
+const flattenPagesData = (
+  data: InfiniteData<IAdminPublications> | undefined
+): IAdminPublicationData[] | undefined => {
+  return data?.pages
+    .map((page: { data: IAdminPublicationData[] }) => page.data)
+    .flat();
+};
+
 const AdminProjectsList = memo(({ className }: Props) => {
+  const { formatMessage } = useIntl();
+  const [search, setSearch] = useState<string>('');
   const { data: authUser } = useAuthUser();
   const isProjectFoldersEnabled = useFeatureFlag({ name: 'project_folders' });
   const userIsAdmin = isAdmin(authUser);
+
   const userIsFolderModerator =
     (authUser &&
       isProjectFoldersEnabled &&
       isProjectFolderModerator(authUser.data)) ??
     false;
   const userCanCreateProject = userIsAdmin || userIsFolderModerator;
   const [containerOutletRendered, setContainerOutletRendered] = useState(false);
+
+  const { data: moderatedAdminPublications } = useAdminPublications({
+    publicationStatusFilter: ['published', 'draft', 'archived'],
+    onlyProjects: true,
+    filter_can_moderate: true,
+    search,
+  });
+
+  const { data: publishedAdminPublications } = useAdminPublications({
+    publicationStatusFilter: ['published'],
+    onlyProjects: true,
+    rootLevelOnly: false,
+    search,
+  });
+
+  const { data: draftAdminPublications } = useAdminPublications({
+    publicationStatusFilter: ['draft'],
+    onlyProjects: true,
+    rootLevelOnly: false,
+    search,
+  });
+
+  const { data: archivedAdminPublications } = useAdminPublications({
+    publicationStatusFilter: ['archived'],
+    onlyProjects: true,
+    rootLevelOnly: false,
+    search,
+  });
+
+  const { data: allAdminPublications } = useAdminPublications({
+    publicationStatusFilter: ['published', 'draft', 'archived'],
+    // Admin publications in the "All" tab are shown in a flat list when there is a search query
+    rootLevelOnly: !search || search.length === 0,
+    search,
+  });
+
   const handleContainerOutletOnRender = (hasRendered: boolean) => {
     setContainerOutletRendered(hasRendered);
   };
 
+  const { pathname } = useLocation();
+
+  const activeTab = getActiveTab(pathname);
+
+  const flatPublishedAdminPublications = flattenPagesData(
+    publishedAdminPublications
+  );
+  const flatDraftAdminPublications = flattenPagesData(draftAdminPublications);
+  const flatArchivedAdminPublications = flattenPagesData(
+    archivedAdminPublications
+  );
+  const flatAllAdminPublications = flattenPagesData(allAdminPublications);
+
+  const flatModeratedAdminPublications = flattenPagesData(
+    moderatedAdminPublications
+  )?.filter((adminPublication) => {
+    // When the user is admin they have access to all projects
```

Comment:
We are working on implementing this filter on the back-end and when done, this filtering behaviour will be deleted from here.